cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -42,7 +42,7 @@ def provision_vm(provider, template):
                                                               template.name)
 
     if not provider.mgmt.does_vm_exist(vm_name):
-        vm.create_on_provider(find_in_cfme=True, allow_skip="default")
+        vm.create_on_provider(find_in_cfme=False, allow_skip="default")
```

Comment:
**Question** Why did you decide to set this to false? Would it not be good to verify that the VM exists before continuing?

Reply:
Originally, this was a workaround for navigation errors. Not relevant anymore.